README.md

```diff
@@ -50,41 +50,41 @@ add your own, or improve on the existing examples, and submit a
 *pull-request*.
 
 To run the examples in your local environment:
+1. Clone the repository:
 
-    1. Clone the repository::
+        git clone https://github.com/pallets-eco/flask-admin.git
+        cd flask-admin
 
-          git clone https://github.com/pallets-eco/flask-admin.git
-          cd flask-admin
+2. Create and activate a virtual environment:
+        
+        virtualenv env -p python3
+        source env/Scripts/activate
```

Comment:
I think we should probably recommend `python3 -m venv venv`?

```suggestion
        python3 -m venv venv
        source venv/Scripts/activate  # Windows
        source venv/bin/activate  # unix
```

Reply:
I agree, so does PSF ).

Reply:
thanks! 🙏